.github/workflows/pre-release-check.yml

```diff
@@ -1,28 +1,30 @@
 name: pre-release-check
 
 env:
-  ENCRYPTION_KEY: ${{ secrets.ENCRYPTION_KEY }}
-  GOOGLE_APPLICATION_CREDENTIALS: scripts/gha-it.json
-  GOOGLE_PROJECT_ID: data-integration-test
-  SBT_OPTS: -DbeamRunners=DataflowRunner
-  DEFAULT_DATAFLOW_ARGS: --runner=DataflowRunner --project=data-integration-test --region=europe-west1  --tempLocation=gs://dataflow-tmp-europe-west1/gha
+  SBT_OPTS: -DbeamRunners=DataflowRunner -Dbigquery.project=data-integration-test
+  DEFAULT_DATAFLOW_ARGS: --runner=DataflowRunner --project=data-integration-test --region=europe-west1 --tempLocation=gs://dataflow-tmp-europe-west1/gha
 
 on:
   workflow_dispatch # Manually triggered
 jobs:
   test-dataflow:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
```

Comment:
it seems that if we use SA key setting permissions is not required?

Reply:
you're right! thanks 👍